Airwallex/AirwallexPayment/Sources/Models/Session.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  Session.swift
+//  Airwallex
+//
+//  Created by Weiping Li on 2025/8/18.
+//  Copyright © 2025 Airwallex. All rights reserved.
+//
+
+#if canImport(AirwallexCore)
+import AirwallexCore
+#endif
+import Foundation
+
+/// `Session` is a specialized subclass of `AWXSession`
+///
+/// This class provides a unified interface for working with the simplified consent flow,
+/// abstracting away the complexity of different payment scenarios (one-off and recurring payments).
+/// It handles both standard payment intents and recurring payment configurations through a
+/// consistent API, making it easier to implement payment processing in your application.
+///
+/// - SeeAlso: `AWXSession`, `PaymentConsentOptions`
+@objc public final class Session: AWXSession {
+    
+    /// The payment intent to handle.
+    @objc public let paymentIntent: AWXPaymentIntent
+    
+    /// Required for recurring payment
+    @objc public let paymentConsentOptions: PaymentConsentOptions?
+    
+    /// Only applicable when payment_method.type is card. If true the payment will be captured immediately after authorization succeeds.
+    /// Default: YES
+    @objc public let autoCapture: Bool
+
+    /// Indicates whether card saving is enabled by default.
+    /// Defaults to YES.
+    @objc public let autoSaveCardForFuturePayments: Bool
+    
+    /// Initialize a new Session
+    /// - Parameters:
+    ///   - countryCode: The country code
+    ///   - paymentIntent: The payment intent to handle
+    ///   - returnURL: Return URL for redirects
+    ///   - applePayOptions: Apple Pay options (optional)
+    ///   - autoCapture: Whether to capture payment immediately after authorization (default: true)
+    ///   - autoSaveCardForFuturePayments: Whether to save card for future payments (default: true)
+    ///   - billing: The billing address (optional)
+    ///   - hidePaymentConsents: Whether to hide stored payment methods (default: false)
+    ///   - lang: The language code (default: system language)
+    ///   - paymentMethods: Array of payment method type names to limit displayed methods (optional)
+    ///   - paymentConsentOptions: Options for recurring payments (optional)
+    ///   - requiredBillingContactFields: Required billing contact fields (default: .name)
+    @objc public init(paymentIntent: AWXPaymentIntent,
+                      countryCode: String,
+                      returnURL: String,
```

Comment:
Since this is not necessary for card payment, I think it can be optional especially when the app hasn't been registered a universal/schema url.